src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -345,6 +353,44 @@ data VoteParam = VoteParam
 instance TypeHasDoc VoteParam where
   typeDocMdDescription = "Describes target proposal id, vote type and vote amount"
 
+------------------------------------------------------------------------
+-- Non FA2
+------------------------------------------------------------------------
+
+data BurnParam = BurnParam
+  { bFrom :: Address
+  , bValue :: Natural
+  , bTokenId :: FA2.TokenId
+  }
+  deriving stock (Generic, Show)
+  deriving anyclass (IsoValue, HasAnnotation)
+
+instance TypeHasDoc BurnParam where
+  typeDocMdDescription = "Describes whose account, which token id and in what amount to burn"
+
+data MintParam = MintParam
+  { mTo :: Address
+  , mValue :: Natural
+  , mTokenId :: FA2.TokenId
+  }
+  deriving stock (Generic, Show)
+  deriving anyclass (IsoValue, HasAnnotation)
```

Comment:
AFAIU in auto-generated `HasAnnotation` field annotations will have `m` prefix, right? In the specification there is no such prefix.